verification/curator-service/ui/src/components/LinelistTable.tsx

```diff
@@ -146,83 +165,91 @@ export default class LinelistTable extends React.Component<{}, LinelistTableStat
+                                    resolve({
+                                        data: flattened_cases,
+                                        page: query.page,
+                                        totalCount: result.data.total,
                                     });
-                                }
-                                resolve({
-                                    data: flattened_cases,
-                                    page: query.page,
-                                    totalCount: result.data.total,
+                                }).catch((e) => {
+                                    this.setState({ error: e.toString() });
+                                    reject(e);
                                 });
-                            }).catch((e) => {
-                                reject(e);
-                            });
-                        })
-                    }
-                    title="COVID-19 cases"
-                    options={{
-                        // TODO: Create text indexes and support search queries.
-                        // https://docs.mongodb.com/manual/text-search/
-                        search: false,
-                        filtering: true,
-                        padding: "dense",
-                        pageSize: 10,
-                        pageSizeOptions: [5, 10, 20, 50, 100],
-                    }}
-                    editable={{
-                        onRowAdd: (newRowData: TableRow) => this.addCase(newRowData),
-                        onRowUpdate: (newRowData: TableRow, oldRowData: TableRow | undefined) =>
-                            this.editCase(newRowData, oldRowData),
-                        onRowDelete: (rowData: TableRow) => this.deleteCase(rowData),
-                    }}
-                />
-            </Paper>
+                            })
+                        }
+                        title="COVID-19 cases"
+                        options={{
+                            // TODO: Create text indexes and support search queries.
+                            // https://docs.mongodb.com/manual/text-search/
+                            search: false,
+                            filtering: true,
+                            padding: "dense",
+                            pageSize: 10,
+                            pageSizeOptions: [5, 10, 20, 50, 100],
+                        }}
+                        editable={{
+                            onRowAdd: (newRowData: TableRow) => this.addCase(newRowData),
+                            onRowUpdate: (newRowData: TableRow, oldRowData: TableRow | undefined) =>
+                                this.editCase(newRowData, oldRowData),
+                            onRowDelete: (rowData: TableRow) => this.deleteCase(rowData),
+                        }}
+                    />
+                </Paper>
+                <div className={classes.error}>{this.state.error}</div>
```

Comment:
I suggest conditional rendering here to avoid that div if there is no error.

```
{this.state.error && <div className={classes.error}>{this.state.error}}
```

https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator

Reply:
Cool! Done.